waiter/src/waiter/handler.clj

```diff
@@ -331,7 +331,7 @@
                                       :unhealthy-instances (-> service-id->unhealthy-instances (get service-id) count)})
           service-id->metrics (service-id->metrics-fn)
           include-effective-parameters? (or (utils/request-flag request-params "effective-parameters")
-                                            (utils/param-contains? request-params "include" "request-info"))
```

Comment:
Why are we removing `request-info` here?

Reply:
This is a copy-paste bug, the flag should be called `effective-parameters`.
I couldn't use the `include=effective-parameters` query parameter as a result in the CLI.